CorsixTH/Lua/map.lua

```diff
@@ -180,8 +180,9 @@ function Map:load(level, difficulty, level_name, map_file, level_intro, map_edit
   local result = file(path .. "Lua" .. pathsep .. "base_config.lua")
 
   local base_config = result
-  local errors
+  local _
```

Comment:
Let's not require declaring (or avoiding declaring) ```_```. It is a near universal convention (mentioned in PIL and enforced by several IDEs) in lua to use _ as a dummy variable so the chance of conflicting with anything else is very slim, which makes this a micro-optimization.